accounts-db/src/account_info.rs

```diff
@@ -124,7 +124,7 @@ impl IsCached for StorageLocation {
 const CACHE_VIRTUAL_STORAGE_ID: AccountsFileId = AccountsFileId::MAX;
 
 impl AccountInfo {
-    pub fn new(storage_location: StorageLocation, lamports: u64) -> Self {
+    pub fn new(storage_location: StorageLocation, is_zero_lamport: bool) -> Self {
```

Comment:
technically breaking public interface, but I doubt we have any consumers of this... also looks like it could be made just crate visible